bundles/org.openhab.transform.rollershutterposition/src/main/resources/OH-INF/i18n/rollershutter.properties

```diff
@@ -0,0 +1,6 @@
+profile.config.rollershutter.position.downtime.label = Down Time
+profile.config.rollershutter.position.downtime.description = Time it takes for roller shutter to extend the full length (in seconds). Defaults to Up Time if not specified.
+profile.config.rollershutter.position.precision.label = Precision
+profile.config.rollershutter.position.precision.description = Minimum movement (in percent) that can be requested. If the requested change is less than this amount, no action will be taken. This may be required for systems where there is a lag in the stop command and consequently it is not possible for fine control of movement. (default = 1)
```

Comment:
Please regenerate this file as some of the strings changed (e.g. default is now 5). See https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file